proto/generate.proto

```diff
@@ -68,6 +73,11 @@ message NextTokenChooserParameters {
     float repetition_penalty = 7;
     /// token watermarking using "A Watermark for Large Language Models"
     bool watermark = 8;
+    bool use_grammar_constraint = 9;
+    string grammar = 10;
+    repeated LogitBias logit_bias = 11;
+    float guidance_scale = 12;
+    string negative_inputs = 13;
```

Comment:
can we add some descriptive comments for each item; similar to the comments above?